lib/net/http.rb

```diff
@@ -942,12 +1028,19 @@ def ipaddr=(addr)
     # Net::WriteTimeout is not raised on Windows.
     attr_reader :write_timeout
 
-    # Maximum number of times to retry an idempotent request in case of
+    # Sets the maximum number of times to retry an idempotent request in case of
     # Net::ReadTimeout, IOError, EOFError, Errno::ECONNRESET,
     # Errno::ECONNABORTED, Errno::EPIPE, OpenSSL::SSL::SSLError,
     # Timeout::Error.
-    # Should be a non-negative integer number. Zero means no retries.
-    # The default value is 1.
+    # The initial value is 1.
+    #
+    # Argument +retries+ must be a non-negative numeric value:
+    #
+    #   http.max_retries = 2   # => 2
+    #   http.max_retries       # => 2
+    #   http.max_retries = 2.9 # => 2.9
+    #   http.max_retries       # => 2
```

Comment:
I know that it's converting it to an integer, but this is a really weird example, let's not encourage people to set floats for these kinds of values 😂

Reply:
Two lines removed from examples.